src/Evolution/Systems/RadiationTransport/M1Grey/M1HydroCoupling.hpp

```diff
@@ -58,22 +58,25 @@ struct ComputeM1HydroCoupling;
  * Compute the source terms of the M1 equation due to neutrino-matter
  * interactions.
  * These are:
+ *
  * \f{align}{
  * \partial_t \tilde E = \alpha W (\sqrt{\gamma} \eta - \kappa_a \tilde J)
  * + \alpha (\kappa_a + \kappa_s) H_n\\
  * \partial_t \tilde S_i = \alpha u_i (\sqrt{\gamma} \eta - \kappa_a \tilde J)
  * - \alpha  (\kappa_a + \kappa_s) H_i.
  * \f}
+ *
  * with \f$W\f$ the Lorentz factor, \f$u_i = W v_i\f$ the spatial components of
  * the fluid 4-velocity, \f$\eta\f$ the emissivity, \f$\kappa_{a,s}\f$
  * the absorption and scattering opacities, \f$J\f$ the comoving energy
  * density, and \f$H_{n,i}\f$ the normal and spatial components of the comoving
  * flux density.
  *
- * The function returns in source_n the energy source and in source_i
- * the momentum source. We write a separate action for these sources to
- * make it easier to switch between implicit / explicit time stepping, as
- * well as to add the source terms to both the fluid and M1 evolutions.
+ * The function returns in \f$source_n\f$ the energy source and in
```

Comment:
sorry we meant surround source_n and source_i  with the character `  which is a tick and will cause doxygen to highlight and monospace .     Unfortunately GitHub interprets ticks also to highlight text...